src/dealDashboard/dealDiscussion/discussionsList/discussionsList.ts

```diff
@@ -35,6 +35,18 @@ export class DiscussionsList{
     intervals: Array(typeof setInterval),
   };
 
+  @computedFrom("deal.clauseDiscussionsV2", "deal.registrationData.terms.clauses")
+  private get discussions(): Map<string, IDealDiscussion> {
+    const discussionsMap = new Map();
+
+    Object.entries(this.deal.clauseDiscussionsV2).forEach(([id, discussion], index) => {
+      discussion.topic = this.deal.registrationData.terms.clauses[index].text;
```

Comment:
Why are we not just using `text` directly?

Why not use DealTokenSwap.clauseDiscussions instead of effectively duplicating it?

Reply:
> `text` directly

done

> effectively duplicating

Currently, the duplication is to set an ens name.
But we could also directly set it to original object.